includes/api/class-edd-api-v2.php

```diff
@@ -40,14 +40,38 @@ public function get_products( $args = array() ) {
 			$query_args = array(
 				'post_type'        => 'download',
 				'posts_per_page'   => $this->per_page(),
-				'suppress_filters' => true,
+				'suppress_filters'  => true,
 				'paged'            => $this->get_paged(),
+				'order'            => $args['order'],
+				'orderby'          => $args['orderby'],
 			);
 
 			if( ! empty( $args['s'] ) ) {
 				$query_args['s'] = sanitize_text_field( $args['s'] );
 			}
 
+			switch ( $query_args['orderby'] ) {
+				case 'price':
+					$query_args['meta_key'] = 'edd_price';
+					$query_args['orderby']  = 'meta_value_num';
+					break;
+
+				case 'sales':
+					if ( user_can( $this->user_id, 'view_shop_sensitive_data' ) || current_user_can( 'view_shop_sensitive_data' ) || $this->override ) {
+						$query_args['meta_key'] = '_edd_download_sales';
+						$query_args['orderby']  = 'meta_value';
```

Comment:
@amdrew shouldn't this also be `meta_value_num` so we can treat them as numeric values?

Reply:
@cklosowski Oops, yes it should. I see how that happened, I copied it over from the downloads shortcode but the shortcode uses `meta_value` for `$atts['orderby']` and `meta_value_num` for the $query['orderby'].

```
case 'sales':
	$atts['orderby']   = 'meta_value';
	$query['meta_key'] = '_edd_download_sales';
	$query['orderby']  = 'meta_value_num';
break;
```

Reply:
Fixed and tested again for peace of mind.